isis/src/base/apps/caminfo/caminfo.xml

```diff
@@ -39,160 +39,167 @@
 Object = Caminfo
   Object = Parameters
     Program     = caminfo
-    IsisVersion = "3.4.2.0 alpha | 2012-08-28"
-    RunDate     = 2012-11-23T18:21:42
-    IsisId      = MeSSEnGeR/MDIS-WAC/1/0250507146:955000
-    From        = EW0250507146G.lev1.cub
-    Lines       = 512
-    Samples     = 512
+    IsisVersion = "3.7.0  | 2019-04-16"
+    RunDate     = 2019-04-24T23:29:22
+    IsisId      = MeSSEnGeR/MDIS-WAC/1/0211286081:889000
+    From        = EW0211286081G.lev1.cub
+    Lines       = 1024
+    Samples     = 1024
     Bands       = 1
   End_Object
 
   Object = Camstats
-    MinimumLatitude   = 57.872627357566
-    MaximumLatitude   = 61.299709008025
-    MinimumLongitude  = 18.065988732773
-    MaximumLongitude  = 24.841561067248
-    MinimumResolution = 229.97591146546
-    MaximumResolution = 232.50589076348
-    MinimumPhase      = 75.229196963297
-    MaximumPhase      = 85.75524278645
-    MinimumEmission   = 0.010416983765103
-    MaximumEmission   = 9.5170724264221
-    MinimumIncidence  = 78.991903468001
-    MaximumIncidence  = 81.783782038128
-    LocalTimeMinimum  = 7.0686934940095
-    LocalTimeMaximum  = 7.5203983163079
+    MinimumLatitude           = 46.17349725649
+    MaximumLatitude           = 54.072584907617
+    MinimumLongitude          = 284.17017788557
+    MaximumLongitude          = 296.47498126014
+    MinimumResolution         = 235.10973750118
+    MaximumResolution         = 238.32543228025
+    MinimumPhase              = 50.107674499885
+    MaximumPhase              = 60.748154060087
+    MinimumEmission           = 0.0076793870352429
+    MaximumEmission           = 11.494912087698
+    MinimumIncidence          = 52.486663651971
+    MaximumIncidence          = 58.235714475896
+    LocalTimeMinimum          = 9.7555337547863
+    LocalTimeMaximum          = 10.575853979758
+    ObliqueResolutionMinimum = 235.23505597394
```

Comment:
Being a little finicky here, but this line is out of alignment.

Reply:
Oops, thanks for catching that!